src/core_plugins/kibana/public/dashboard/components/full_screen_placeholder.js

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import chrome from 'ui/chrome';
+
+import {
+  KuiKeyboardAccessible,
+} from 'ui_framework/components';
+import {
+  keyCodes,
+} from 'ui_framework/services';
+
+export class FullScreenModePlaceholder extends PureComponent {
+
+  onKeyDown = (e) => {
+    if (e.keyCode === keyCodes.ESCAPE) {
+      this.props.onExitFullScreenMode();
+    }
+  };
+
+  componentWillMount() {
+    document.addEventListener('keydown', this.onKeyDown, false);
+    chrome.setVisible(false);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.onKeyDown, false);
+    chrome.setVisible(true);
+  }
+
+  render() {
+    return (
+      <div
+        className="fullScreenModePlaceholder"
+      >
+        <KuiKeyboardAccessible>
+          <div
```

Comment:
Can we make this element a `button` instead of a `div`? It might require a couple more styles on the `.exitFullScreenMode` class to remove the default button styling, but it means we won't need `KuiKeyboardAccessible` here.